src/main/java/bio/terra/service/dataset/flight/create/DatasetCreateFlight.java

```diff
@@ -20,6 +20,8 @@
 import bio.terra.stairway.RetryRuleExponentialBackoff;
 import org.springframework.context.ApplicationContext;
 
+import static bio.terra.common.FlightUtils.getDefaultExponentialBackoffRetryRule;
```

Comment:
Is there Jade consensus on using static imports? My preference is to avoid it, except in test code where it's handy to write asserts, matchers, etc. In other places I find it makes the code harder to read.

Reply:
I don't know that there's consensus.  I tend to like this way since it makes the lines shorter

Reply:
Honestly, I have not paid attention. I just let IntelliJ have its way with the imports.
Which is to say, I don't have an opinion.